pkg/core/pipeline/source/main.go

```diff
@@ -179,22 +184,18 @@ func (s *Source) Unmarshal() (sourcer Sourcer, changelog Changelog, err error) {
 		}
 
 	case "githubRelease":
-		githubSpec := github.Spec{}
+		var sourceSpec github.Spec
 
-		err := mapstructure.Decode(s.Config.Spec, &githubSpec)
-
-		if err != nil {
+		if err := mapstructure.Decode(s.Config.Spec, &sourceSpec); err != nil {
 			return nil, nil, err
 		}
 
-		g, err := github.New(githubSpec)
-
+		grPrResource, err := githubrelease.New(sourceSpec)
```

Comment:
What's the meaning of the variable grPrResource?

Reply:
Good catch, it is a typo. Should be the following:

```suggestion
		ghrResource, err := githubrelease.New(sourceSpec)
```

WDYT?

Reply:
I went for `newGhrResource`. WDYT?